app/api/user/schemas.py

```diff
@@ -15,3 +18,31 @@ class User(UserBase):
 
     class Config:
         orm_mode = True
+
+
+class UserUpdate(BaseModel):
+    email: EmailStr
+    username: constr(min_length=1, strip_whitespace=True)
```

Comment:
```suggestion
    email: Union[EmailStr, None]
    username: Union[constr(min_length=1, strip_whitespace=True), None]
```

`email` 和 `username` 也是 optional 的